validation/src/test/scala/hmda/validation/rules/lar/validity/V290Spec.scala

```diff
@@ -0,0 +1,40 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class V290Spec extends LarEditCheckSpec {
+
+  property("Succeeds when valid MSA/MD, state and county codes are present") {
+    forAll(larGen) { lar =>
+      val validGeography = lar.geography.copy(msa = "17020", county = "007", state = "06")
+      val validLar = lar.copy(geography = validGeography)
+      V290(validLar) mustBe a[Success]
+    }
+  }
+
+  property("Succeeds when MSA == NA") {
+    forAll(larGen) { lar =>
+      val validGeography = lar.geography.copy(msa = "NA")
+      val validLar = lar.copy(geography = validGeography)
+      V290(validLar) mustBe a[Success]
+    }
+  }
+
+  property("Succeeds when code is MD (not MSA) with valid state and county combination") {
+    forAll(larGen) { lar =>
+      val validGeography = lar.geography.copy(msa = "48424", state = "12", county = "086")
+      val validLar = lar.copy(geography = validGeography)
+      V290(validLar) mustBe a[Success]
```

Comment:
I want to make sure I understand the business logic correctly.  It looks like, if the LAR has an MD instead of an MSA, then it's OK to have a state+county combination that matches anything within the MSA that contains that MD -- even if the county is outside of the given MD.

So in this example, we have MD 48424, which is for county 099 (Palm Beach County), in state 12 (Florida).  Its larger MSA is 33100, which also includes counties 011 (Broward) and 086 (Miami-Dade), in their own separate MDs.

This test says that it's OK for a LAR to have MD 48424 with state 12 and county 086, because the MSA matches, even though the specific MD doesn't.  Am I understanding correctly, and is this the intended business logic?

If so, then please:
 1. add a test for the "normal" case of `12|086|48424`, and
 2. maybe say something a bit more explicit in the description of this test, because it took me a while to figure out what it meant.

If not, then let's discuss here or in real-time chat tomorrow, so I can understand the requirement and finish the review.